pilot/docker/Dockerfile.proxy_debug

```diff
@@ -36,5 +36,8 @@ RUN useradd -m --uid 1337 istio-proxy && \
     echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers && \
     chown -R istio-proxy /var/lib/istio
 
+# Control tracing using legacy runtime API
+COPY random_sampling /etc/istio/proxy/runtime/tracing/random_sampling
```

Comment:
maybe that can be a configmap mount ?

Reply:
config map sounds good.
Will make that a separate PR.